src/main/webapp/app/views/admin/info/edit/input-text_with_definition.html

```diff
@@ -1,5 +1,5 @@
 <div class="input-group input-text-with-definition">
```

Comment:
Where is this input being used?

Reply:
It is not. 

Here is the InputType: https://github.com/TexasDigitalLibrary/Vireo/blob/4.0.x/src/main/resources/input_types/SYSTEM_Input_Types.json#L27

Here is the admin template: https://github.com/TexasDigitalLibrary/Vireo/blob/4.0.x/src/main/webapp/app/views/admin/info/edit/input-text_with_definition.html

Here is the student template: https://github.com/TexasDigitalLibrary/Vireo/blob/4.0.x/src/main/webapp/app/views/inputtype/input-text_with_definition.html

It is not used in the initial field profiles anywhere.

https://github.com/TexasDigitalLibrary/Vireo/blob/4.0.x/src/main/resources/organization/SYSTEM_Organization_Definition.json

Simply removing from default Input_Tyes.json and deleting the two templates is all that is needed to remove this relic.